src/main/scala/org/scalacheck/Cogen.scala

```diff
@@ -80,8 +80,9 @@ object Cogen extends CogenArities with CogenLowPriority with CogenVersionSpecifi
     @tailrec
     def normalize(unscaled: BigInteger, scale: Int): (BigInteger, Int) = {
       val divideAndRemainder = unscaled.divideAndRemainder(BigInteger.TEN)
-      val quotient = divideAndRemainder(0)
-      val remainder = divideAndRemainder(1)
+      // java.math.BigInteger.divideAndRemainder | IM | returned array content should not be null.
```

Comment:
**Java classification**
java.math.BigInteger.divideAndRemainder | IM
Returned Array[T | Null], contents of array are non-nullable.